src/main.js

```diff
@@ -3438,7 +3430,7 @@ function loadSnapshotDragAndDropSetup() {
     }
 };
 
-function loadSnapshotPreflight() {
+window.loadSnapshotPreflight = () => {
     var files = document.getElementById('loadSnapshotSelectFiles').files;
     if (files.length <= 0) {
```

Comment:
could ```files.length``` ever be less then 0? 
May be we can replace with if(!file.length)?

Reply:
yes, definitely :) it was the old code